Documentation/TopLevelObjects/Config.rst

```diff
@@ -554,6 +554,19 @@ Properties of 'config'
         Modify the whole :html:`<html>` tag with stdWrap functionality. This can be
         used to extend or override this tag.
 
+        For example it can be used to distinctly set a :html:`lang` attribute,
+        that may diverge from the automatically created attribute using the site
+        language settings. Normally, :html:`lang` is set to something like `en`
+        or `en`, only indicating the primary language. You may want to use the
```

Comment:
like "en" or "en"? Do you mean "de" or "en"?

Reply:
```suggestion
        or `de`, only indicating the primary language. You may want to use the
```